app/lib/aether/languages/coffeescript.coffee

```diff
@@ -0,0 +1,17 @@
+_ = window?._ ? self?._ ? global?._ ? require 'lodash'  # rely on lodash existing, since it busts CodeCombat to browserify it--TODO
```

Comment:
These languages don't seem to exist and I should probably try to delete them.
The issue is that the method `useFunctionWrapping` is required in the main code flow even if the rest isn't.